clang-tools-extra/unittests/clang-doc/HTMLGeneratorTest.cpp

```diff
@@ -1,4 +1,4 @@
-//===-- clang-doc/HTMLGeneratorTest.cpp -----------------------------------===//
+//===-- clang-doc/HTMLGeneratorTest.cpp ---------------------------===//
```

Comment:
Let's be consistent. If we're removing the file name from the header elsewhere, lets do it here to. I'm also fine w/ not modifying the headers to minimize the diff.